docs/cue/reference/components/sinks.cue

```diff
@@ -260,11 +260,13 @@ components: sinks: [Name=string]: {
 											required:    false
 											type: float: default: 0.7
 										}
-										rtt_threshold_ratio: {
-											common:      false
-											description: "When comparing the past RTT average to the current measurements, we ignore changes that are less than this ratio higher than the past RTT. Valid values are greater than or equal to 0. Larger values cause the algorithm to ignore larger increases in the RTT."
-											required:    false
-											type: float: default: 0.05
+										rtt_deviation_scale: {
```

Comment:
I would be quite open to a better name for this configuration. The term "deviation" has too many bad connotations, but I wasn't coming up with anything better that was still descriptive.

Reply:
I'm fine with deviation as a word, used here in a technical sense. That said, considering this it strikes me that having a measurement of the skew of the distribution would be interesting as well.

Reply:
Agreed, it would be interesting, but I'm not sure how useful in the end. Also, I have no idea how to even calculate that given the EWMA scheme in use.

Reply:
Fair point. More an idle thought than anything.